### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by bharathkkb**
minor fixes for errors I found during walkthrough

### CODE REVIEW

File: 05-Load-Balancer/main.tf [hunk 1]
```diff
@@ -26,6 +26,7 @@
  *     - port_name: "http"
  *     - protcol: "HTTP"
  *     - port: 80
+ *     - port_name: "httpport"
```
Comment (marine675) [inline 05-Load-Balancer/main.tf]: line 26 already a `port_name`?

**Review by marine675 — CHANGES_REQUESTED** [review]
duplicate `port_name`

File: 05-Load-Balancer/main.tf (hunk 1 above)
Comment (bharathkkb) [reply 05-Load-Balancer/main.tf]: wow I totally didn't see that :D

**Review by marine675 — APPROVED** [review]
LGTM

**Pull request merged**